pkg/services/accesscontrol/database/database.go

```diff
@@ -92,7 +92,8 @@ func (s *AccessControlStore) SearchUsersPermissions(ctx context.Context, orgID i
 					) AS sa ON 1 = 1
 					WHERE br.role = ?
 		) AS up
-		WHERE (org_id = ? OR org_id = ?)
+		INNER JOIN role AS r on up.role_id = r.id
```

Comment:
Oh I understand now why you preferred the other approach! It adds an additional join :confounded:

Reply:
Yes, one little detail :) We actually need it only when filter by role name, so I'll try do add it under condition.